test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python manage.py test auth_backends --traceback --failfast
```

Comment:
Why are there two scripts for running tests? You should be able to merge them both:

```shell
python manage.py test auth_backends --with-coverage --cover-package=auth_backends
```